frontend/src/console/components/CoursesManage/__snapshots__/CoursesManage.spec.tsx.snap

```diff
@@ -241,11 +241,9 @@ exports[`renders without crashing 1`] = `
                         General
                       </a>
                       <a
-                        aria-current="page"
-                        className="disabled active"
-                        href="/"
+                        aria-current={null}
```

Comment:
@shimulch The `NavLink` component sets the aria-current property and does not have a prop to update it.
Also, the other  `NavLink`s above render the `aria-current` as `null` for testing.

I think fixing the snapshots can be a separate task as this will require updates in many places